scapy/sendrecv.py

```diff
@@ -127,7 +128,12 @@ def sndrcv(pks, pkt, timeout = None, inter = 0, verbose=None, chainCC=0, retry=0
                                 if len(inp) == 0 or pks in inp:
                                     r = pks.nonblock_recv()
                             else:
-                                inp, out, err = select(inmask,[],[], remaintime)
+                                inp = []
+                                try:
+                                    inp, out, err = select(inmask,[],[], remaintime)
+                                except Exception,e:
```

Comment:
Thanks! Could you fix the code to make it more compliant to the standards (no blank line, space after comma, etc.)?

```Python
                                except Exception, exc:
                                    if exc.errno != errno.EINTR:
                                        raise
                                if len(inp) == 0: 
```